dependencies.gradle

```diff
@@ -20,6 +21,18 @@ ext {
                     core            : [
                             ktx: "androidx.core:core-ktx:1.2.0"
                     ],
+                    activity        : [
+                            compose : "androidx.activity:activity-compose:1.3.0-alpha05"
+                    ],
+                    compose         : [
+                            activity    : "androidx.activity:activity-compose:1.3.0-alpha05",
+                            foundation  : "androidx.compose.foundation:foundation:$composeVersion",
+                            material    : "androidx.compose.material:material:$composeVersion",
```

Comment:
why would you need material in formula ?

Reply:
It's used in `samples/stopwatch-compose`